proxy.sh

```diff
@@ -47,6 +47,19 @@ pids="${pids} $!"
 &
 pids="${pids} $!"
 
+# Start the resource server:
+./oran-o2ims start resource-server \
+--log-file="servers.log" \
+--log-level="debug" \
+--log-field="server=resource" \
+--log-field="pid=%p" \
+--api-listener-address="127.0.0.1:8002" \
+--cloud-id="123" \
+--backend-url="${BACKEND_URL}" \
```

Comment:
Not related to this PR, just wonder if we should support multiple backend URLs?
I.e. should we open a ticket for supporting something like RESOURCE_BACKEND_URL, in case the search api is routed using a different domain? Or is it an unlikely scenario...

Reply:
Yes, you are right, we should support multiple backend URLs, one for each kind of backend. Please open that ticket.